test/Microsoft.AspNet.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -0,0 +1,698 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
This was taken directly from MVC since rendering is now handled by `TagHelperOutput` instead of a method on RazorPage.